openapi/openapi.json

```
@@ -27048,10 +27048,22 @@
           "x-kubernetes-list-type": "map"
         },
         "customLogoFile": {
-          "description": "customLogoFile replaces the default OpenShift logo in the masthead and about dialog. It is a reference to a ConfigMap in the openshift-config namespace. This can be created with a command like 'oc create configmap custom-logo --from-file=/path/to/file -n openshift-config'. Image size must be less than 1 MB due to constraints on the ConfigMap size. The ConfigMap key should include a file extension so that the console serves the file with the correct MIME type. Recommended logo specifications: Dimensions: Max height of 68px and max width of 200px SVG format preferred",
+          "description": "customLogoFile replaces the default OpenShift logo in the masthead and about dialog. It is a reference to a ConfigMap in the openshift-config namespace. This can be created with a command like 'oc create configmap custom-logo --from-file=/path/to/file -n openshift-config'. Image size must be less than 1 MB due to constraints on the ConfigMap size. The ConfigMap key should include a file extension so that the console serves the file with the correct MIME type. Recommended logo specifications: Dimensions: Max height of 60px and max width of 200px SVG format preferred Deprecated: Use CustomLogos instead.",
           "default": {},
           "$ref": "#/definitions/com.github.openshift.api.config.v1.ConfigMapFileReference"
         },
+        "customLogos": {
+          "description": "customLogos is an optional field that can be used to replace the OpenShift Masthead and Favicon logos in the console UI with custom logos. Each entry must have a unique logo type. When specified, there must be at least one entry and must not have greater than 2 entries. Each item in the list must have a unique Type.",
```

A couple tweaks for clarity
```suggestion
          "description": "customLogos is an optional field that can be used to replace the OpenShift Masthead and Favicon logos with custom logos in the console UI. Each entry must have a unique logo type. When specified, there must be at least one entry, and there must not be more than 2 entries. Each item in the list must have a unique Type.",
```